CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - in case of vulnerabilities
 
+## [1.1.1] - 2019-08-13
```

Comment:
Need to update the link(s) at the bottom of the file so this [1.1.1] points to something.
I was going to propose a quick edit, but GitHub is fighting with me about being on a branch in order to propose changes, blah blah